test/functional/tool_wallet.py

```diff
@@ -54,12 +54,7 @@ def assert_tool_output(self, output, *args):
         assert_equal(p.poll(), 0)
 
     def wallet_shasum(self):
```

Comment:
not sure if we'd want to change the name of the test aswell to `wallet_sha256sum` to be more accurate

Reply:
Thanks for reviewing! Decided to keep the name as it is, as sha256 falls into the SHA message digests category (see e.g. https://linux.die.net/man/1/shasum) and for the sake of the test concrete type of hash used doesn't matter anyway. All we care is detecting if the wallet file contents have been changed.